src/track/keyutils.cpp

```diff
@@ -24,39 +21,39 @@ static const QString s_lancelotKeyPattern("^\\s*(1[0-2]|[1-9])([ab])\\s*$");
 // anchor the pattern so we don't get accidental sub-string matches
 // (?:or)? allows unabbreviated major|minor without capturing
 static const QString s_keyPattern = QString::fromUtf8(
-        "^\\s*([a-g])([#♯b♭]*)"
+        "^\\s*([a-g])([#" MUSIC_SHARP_SIGN_UTF8 "b" MUSIC_FLAT_SIGN_UTF8
```

Comment:
Could this be the issue? If you use `\uXXXX` notation in a regex, doesn't the `\` need to be escaped?

Reply:
Good catch. First we need a regression test.

Reply:
Nope. We already have test for the sharp/flat characters and they succeed. There are no escapes in these strings. Plain UTF-8 that is converted to UTF-16.

Reply:
The expressions are equal. It doesn't matter if you compose them from an UTF-8 character or the corresponding byte sequence.

Reply:
The issues with editing traditional keys with Unicode characters are already present in the current version without this PR. Unrelated.